.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+      - master
+      - patch-1
```

Comment:
Any reason to include `patch-1`?
Include `photon-style` instead.

Reply:
I did that for a temporary check. I'll delete it right away.